usr/share/rear/restore/GALAXY11/default/500_migrate_commvault_certificates.sh

```diff
@@ -0,0 +1,8 @@
+# If commvault agent certificates are not copied, the agent will not work on the restored system
+
+local commvault_base_dir=/opt/commvault/Base64
+local certs=$commvault_base_dir/certificates
+# copy certificates to the restored filesystem
+if type -r $certs; then
```

Comment:
@schlomo 
There is no option for -r. I think the type command is wrong here.
Should the existence of the directory be checked here?

Maybe like this?
` if [ -d "$certs" ]; then`

Reply:
Yes, you are right. Thank you!